microcosm_pubsub/metrics.py

```diff
@@ -120,3 +120,52 @@ def __call__(self, elapsed_time: float, message_batch_size: int):
             message_batch_size,
             tags=tags,
         )
+
+
+@defaults(
+    enabled=typed(boolean, default_value=True)
+)
+class PubSubProducerMetrics:
+    """
+    Send metrics regarding the producer
+
+    """
+
+    def __init__(self, graph):
+        self.metrics = self.get_metrics(graph)
+        self.enabled = bool(
+            self.metrics
+            and self.metrics.host != "localhost"
+            and graph.config.pubsub_send_metrics.enabled
```

Comment:
Should this be pubsub_producer_metrics.enabled?

Reply:
No. This is the generic one that we send in ansible.
I don't want to have a fine grained one. If we track pubsub metrics, it should track dispatcher and producer.

Reply:
Sounds good.